packages/block-library/src/term-description/editor.scss

```diff
@@ -0,0 +1,4 @@
+.wp-block-term-description__placeholder {
+	padding: 1em 0;
```

Comment:
This one (padding) should be removed probably so as to match the end result in front end.

Reply:
Other placeholders (like for example the post-content have a `height:100px` for the placeholder, and then add `display:flex` to it. This makes the placeholder text vertically align inside its container and the dashed borders of placeholders don't touch the text, as there is a gap between them.
In this case display:flex and height:100px could not be used. To compensate for that, vertical padding was added to the placeholder 😉 
It's a reasonable compromise, and ideally all placeholders would be refactored in a future PR. What we have now doesn't quite work if we start changing font-sizes etc...

Reply:
I'm not sure I understand this 100%. If we just removed the padding we would have other possible styling issues? Because I tried it through dev tools only and seemed identical to front-end. Of course I tested with one theme (tt1-blocks) and you're certainly more experienced in themes :) 

It's a small thing that can be changed in the future if we need to, but want to understand better :)